sys/include/net/ble.h

```diff
@@ -85,6 +85,39 @@ extern "C" {
 #define BLE_DESC_VALUE_TRIGGER_SETTING      (0x290a)
 /** @} */
 
+/**
+ * @name    Selected GATT service UUIDs (16-bit)
+ *
+ * @see https://www.bluetooth.com/specifications/gatt/services
+ * @{
+ */
+#define BLE_GATT_SVC_GAP                    (0x1800)    /**< GAP service */
+#define BLE_GATT_SVC_GATT                   (0x1801)    /**< GATT service */
+#define BLE_GATT_SVC_DEVINFO                (0x180a)    /**< device info */
+#define BLE_GATT_SVC_HRS                    (0x180d)    /**< heart rate service */
+#define BLE_GATT_SVC_BAS                    (0x180f)    /**< battery service */
+#define BLE_GATT_SVC_IPSS                   (0x1820)    /**< IP protocol support */
+/* add more on demand */
+/** @} */
+
+/**
+ * @name    Selected GATT characteristic UUIDs (16-bit)
+ *
+ * @see https://www.bluetooth.com/specifications/gatt/characteristics/
+ * @{
+ */
+#define BLE_GATT_CHAR_BATTERY_LEVEL         (0x2a19)    /**< battery level */
+#define BLE_GATT_CHAR_SYSTEM_ID             (0x2a23)    /**< system ID */
+#define BLE_GATT_CHAR_MODEL_NUMBER_STR      (0x2a24)    /**< model number */
+#define BLE_GATT_CHAR_SERIAL_NUMBER_STR     (0x2a25)    /**< serial number */
+#define BLE_GATT_CHAR_FW_REV_STR            (0x2A26)    /**< firmware revision */
```

Comment:
For consistency, maybe use lower case `A` in hex value ?

Reply:
yes, copy-paste typo, will fix.